### DISCUSSION TITLE
right click on mac seems not to work

### DISCUSSION THREAD

**Discussion by oaklandpaul** [cables-gl/cables_docs#587] [Answers & Questions]
in other applications the right click works, just not in cables.gl.  i've tried chrome, firefox and also resetting computer. thanks much for any help.

**Comment by pandrr** [comment]
do you mean an op or the cables editor itself ? do you use a mouse or a touchpad ? what are you trying to right click ?

**Comment by oaklandpaul** [comment]
inside the cables editor. i'm using a touchpad and right clicking with
control button.

i'm trying to do cable unplugging and plugging elsewhere but the behavior
is like a left mouse click.

thanks much,

paul

**Comment by pandrr** [comment]
it works with two fingers. we recommend a mouse for serious patching

**Comment by oaklandpaul** [comment]
ah yes that worked! thanks so much!

will look into mouse for better patching.

paul
